graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -88,19 +89,15 @@ def get_superclasses_from_class_definition(class_definition):
     return []
 
 
+@six.add_metaclass(ABCMeta)
```

Comment:
Tested that this works as expected for python2/python3. In other words, tested that we get an error if we try to instantiate a class with this decorator, but we don't if we instantiate a subclass that overrides its abstract methods.